lib/node_modules/@stdlib/array/base/mskfilter-map/docs/types/index.d.ts

```diff
@@ -0,0 +1,171 @@
+
+/**
+* Callback invoked for each array element.
+*
+* @param value - array element
+* @returns result
+*/
+type Unary<T, U, V> = ( this: V, value: T ) => U;
+
+/**
+* Callback invoked for each array element.
+*
+* @param value - array element
+* @param indices - current array element indices
+* @returns result
+*/
+type Binary<T, U, V> = ( this: V, value: T, indices: Indices ) => U;
+
+/**
+* Callback invoked for each array element.
+*
+* @param value - array element
+* @param indices - current array element indices
+* @param array - input array
+* @returns result
+*/
+type Ternary<T, U, V> = ( this: V, value: T, indices: Indices, array: Array2D<T> ) => U;
+
+/**
+* Callback invoked for each array element.
+*
+* @param value - array element
+* @param indices - current array element indices
+* @param array - input array
+* @returns result
+*/
+type Callback<T, U, V> = Nullary<U, V> | Unary<T, U, V> | Binary<T, U, V> | Ternary<T, U, V>;
+
+/**
+* Interface describing the main export.
+*/
+interface Routine {
+	/**
+	* Applies a mask to a provided input array and returns a new array after applying a mapping function.
+	*
+	* @param x - input array
+	* @param mask - mask array
+	* @param clbk - callback to invoke
+	* @param thisArg - execution context
+	* @returns output array
+	*
+	* @example
+	* var x = [ 1, 2, 3, 4 ];
+	*
+	* function scale( val ) {
+	* 	return 10 * val;
+	* }
+	*
+	* var y = mskfilterMap( x, [ 0, 1, 0, 1 ], scale );
+	* // returns [ 20, 40 ]
+	*/
+	<T = unknown, U = unknown, V = unknown>( x: Collection<T>, mask: Collection<T>, clbk: Callback<T, U, V>, thisArg?: ThisParameterType<Callback<T, U, V>> ): Array<U>;
+
+	/**
+	* Applies a mask and mapping function to an indexed array and assigns results to elements in an indexed output array.
+	*
+	* @param x - input nested array
+	* @param mask - mask array
+	* @param out - output array
+	* @param stride - output array stride
+	* @param offset - output array offset
+	* @param clbk - callback to invoke
+	* @param thisArg - execution context
+	* @returns output array
+	*
+	* @example
+	* var x = [ 1, 2, 3, 4 ];
+	* var mask = [ 0, 1, 0, 1 ];
+	*
+	* var out = [ 0, 0, 0, 0 ];
+	*
+	* function scale( x ) {
+	*     return x * 10;
+	* }
+	*
+	* var arr = mskfilterMap.assign( x, mask, out, 1, 0, scale );
+	* // returns [ 20, 40, 0, 0 ]
+	*
+	* var bool = ( arr === out );
+	* // returns true
+	*/
+	assign<T = unknown, U = unknown, V = unknown>( x: Collection<T>, mask: Collection<T>, out: Collection<T>, stride: number, offset: number, clbk: Callback<T, U, V>, thisArg?: ThisParameterType<Callback<T, U, V>> ): Array<U>;
+}
+
+/**
+* Applies a mask to a provided input array and returns a new array after applying a mapping function.
+*
+* @param x - input array
+* @param mask - mask array
+* @param clbk - callback to invoke
+* @param thisArg - execution context
+* @returns output array
+*
+* @example
+* var x = [ 1, 2, 3, 4 ];
+*
+* function scale( val ) {
+* 	return 10 * val;
```

Comment:
```suggestion
* 	return 10 * val;
```

You have TAB indentation here, rather than spaces.